plugins/outputs/amqp/amqp.go

```diff
@@ -3,7 +3,6 @@ package amqp
 import (
 	"bytes"
 	"fmt"
-	"log"
```

Comment:
we might want our own linter that tells users not to import log into plugins.

Reply:
We can just enable and configure [imports-blacklist](https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md#imports-blacklist) rule for that :)